util/topgen.py

```diff
@@ -603,10 +607,12 @@ def generate_top_ral(top: Dict[str, object], name_to_block: Dict[str, IpBlock],
         if t not in inst_to_block.values():
             del name_to_block[t]
 
-    chip = Top(regwidth, name_to_block, inst_to_block, if_addrs, mems, attrs)
+    addr_spaces = {addr_space["name"] for addr_space in top["addr_spaces"]}
+    chip = Top(regwidth, addr_spaces, name_to_block, inst_to_block, if_addrs, mems, attrs)
 
     # generate the top ral model with template
-    return gen_dv(chip, dv_base_names, str(out_path))
+    gen_dv(chip, dv_base_names, str(out_path))
```

Comment:
You cannot ignore the return value from gen_dv: non-zero indicates an error.

Reply:
Changed that back to the original code.

Reply:
👍